Makefile

```diff
@@ -7,17 +7,22 @@ EF_TESTS_DIR := ./crates/ef-testing/ethereum-tests
 
 # Downloads and unpacks Ethereum Foundation tests in the `$(EF_TESTS_DIR)` directory.
 # Requires `wget` and `tar`
+.PHONY:$(EF_TESTS_DIR)
```

Comment:
do we really want the `.PHONY` ? see https://hackernoon.com/what-does-phony-do-in-a-makefile

Reply:
Indeed will remove